webapp/components/interop-data.js

```diff
@@ -16,7 +16,7 @@ export const interopData = {
         'score_as_group': false
       }
     ],
-    'csv_url': 'https://raw.githubusercontent.com/web-platform-tests/results-analysis/gh-pages/data/interop-2021/interop-2021-{stable|experimental}-v2.csv',
+    'csv_url': '/static/interop-2021-{stable|experimental}-v2.csv',
```

Comment:
I am thinking we should have a comment or note in the docs on where we get this CSV from in the future.
Previously, we had the direct link, so a developer would know exactly where to get next year's CSV.
Now, that we are shipping it with our code, a question a future developer would have is: Do I need to generate this CSV? If so, how? (But the answer would be to refer to the results-analysis repo.)